docs/Interoperator-authentication.md

```diff
@@ -0,0 +1,56 @@
+# Service Fabrik Inter-operator Authentication
+
+##  Abstract
+
+This document describes the basic and mTLS authentication supported by the Service Fabrik inter-operator broker.
+
+## Table of Content
+- [Service Fabrik Inter-operator Basic Architecture](#service-fabrik-inter-operator-basic-architecture)
+  - [Abstract](#abstract)
+  - [Table of Content](#table-of-content)
+  - [Context](#context)
+  - [BasicAuthentication](#basic-authentication)
+  - [mTLSAuthentication](#out-of-the-box-mtls-authentication)
+    - [CertificateIdentityVerification](#certificate-identity-verification)
+
+## Context
+The SAP Service Manager supports two authentication mechanisms for communicating with the brokers:
+  - Basic Authentication: validates the username and password.
+  - mTLS Authentication: validates the client certificate's subject pattern.
+
+## Basic Authentication
+To use Basic Authentication, the username and password should be provided during the registration of the broker in the Service Manager.
```

Comment:
Hi @Pooja-08, This document is specific to SAP and SAP's service manager... might be you can move service manager related mentions to internal github page?